regress/expected/cypher_match.out

```diff
@@ -889,6 +889,41 @@ $$) AS (i agtype);
  {"id": 1688849860263938, "label": "v2", "properties": {"id": "middle"}}::vertex
 (3 rows)
 
+-- 
```

Comment:
There needs to be test cases for when Optional Match is the first clause in a cypher statement.

Reply:
OK. Added more tests.